src/main/java/tetris/Tetromino.java

```diff
@@ -198,4 +198,36 @@ public Tetromino rotateClockwise(Tetromino shape) {
     }
 
     //add rotateCounter()
+
+    /**
+     * @return last element/bottom of shape.
+     * Used if the shape reached bottom of grid or other shape at bottom.
+     */
+    public Square getShapeBottom(Tetromino shape) {
```

Comment:
This is in Tetromino so doesn't need a parameter

Reply:
i changed it to getShapeBottom(List<Square> shape) bc I was thinking we were going to need to see what shape the current shape that's falling is but im realizing the entire method should be changed and i should make a getCurrentShape method

Reply:
*getShapeBottom(List<Square> shape)